src/Sylius/Bundle/UiBundle/Resources/assets/less/AdminLTE/skin-sylius.less

```diff
@@ -0,0 +1,71 @@
+@import "node_modules/bootstrap/less/mixins.less";
```

Comment:
I know, but I was wondering why the folder `assets` name ? My suggestion was to use a folder named `private` (not published) in opposition to `public` (published).
That's minor but I think it would be easier to understand if we have something like: 

```
*Bundle/
    Resources/
        public/
        private/
            sass/
            css/
        config/
```